it/src/test/java/com/linecorp/centraldogma/it/mirror/git/GitMirrorTest.java

```diff
@@ -416,6 +467,7 @@ private void pushMirrorSettings(String localRepo, @Nullable String localPath, @N
                                         "  \"localRepo\": \"" + localRepo + "\"," +
                                         (localPath != null ? "\"localPath\": \"" + localPath + "\"," : "") +
                                         "  \"remoteUri\": \"" + gitUri + firstNonNull(remotePath, "") + '"' +
+                                        ",\"remoteExclude\": \"" + firstNonNull(remoteExclude, "") + "\"" +
```

Comment:
How about using `gitIgnore` instead of `remoteExclude` so that users easily notice what it is?
Also we don't have to change the field name when we implement CD_TO_GIT mirroring. 😉